discord-scripts/fix-linear-embed.ts

```diff
@@ -13,6 +13,16 @@ const { LINEAR_API_TOKEN } = process.env
 
 // track processed message to avoid duplicates if original message is edited
 const processedMessages = new Map<string, Set<string>>()
+// let us also track sent embeds to delete them if the original message is deleted or edited WIP
+const sentEmbeds = new Map<string, Message>()
+
+const ISSUE_PREFIXES = ["THESIS", "MEZO", "ENG"]
```

Comment:
Can we resolve these from the Linear API?

Reply:
Yeah good call! Was thinking the same we can pull this in initially to populate from sdk. Otherwise, working through  some edge case / hurdles to sort out on edits & handling multiple issues in one after edits.